README.md

```diff
@@ -1,21 +1,54 @@
-# prismarine-template
+# XboxLive-Auth
 [![NPM version](https://img.shields.io/npm/v/prismarine-template.svg)](http://npmjs.com/package/prismarine-template)
 [![Build Status](https://github.com/PrismarineJS/prismarine-template/workflows/CI/badge.svg)](https://github.com/PrismarineJS/prismarine-template/actions?query=workflow%3A%22CI%22)
 [![Discord](https://img.shields.io/badge/chat-on%20discord-brightgreen.svg)](https://discord.gg/GsEFRM8)
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-template)
 
-A template repository to make it easy to create new prismarine repo
+Quickly and easily obtain an xbox token to authenticate with Minecraft/Mojang
+
+## Installation
+```shell
```

Comment:
What do you want to name it?

Reply:
> I just realized this conflicts with the Xbox replay lib with the same name, let’s try and avoid that

it's not a conflict since they are name scoped

Reply:
Yes I get that, but it seems easy to mistake one lib for the other with the same name. Maybe something like xbl-auth or prismarine-xboxlive-auth ? Maybe @rom1504 has some input?